### DISCUSSION TITLE
How to achieve load balancing when multiple workers listen to the same queue?

### DISCUSSION THREAD

**Discussion by StringKe** [taskforcesh/bullmq#1230] [Q&A]
I have a queue that stores all tasks, and I hope that the workers listening to this queue can process the tasks separately.

**Comment by manast** [comment]
So whats the problem?

**Reply by StringKe** [reply]
When the job is executed, it is considered successful and I wish I could control it manually.

**Reply by StringKe** [reply]
I try to use bullmq with socketio .
A worker is also created when socketio has a new connection.
The status of the job is controlled by the events returned by socketio, rather than when the job is consumed, the job is considered complete.

I will get this error `Error: Missing lock for job 5LvCVu43yzm8KCT-wCGim.finished`
